include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -117,6 +117,10 @@ ORT_DEFINE_RELEASE(ModelMetadata);
 ORT_DEFINE_RELEASE(ThreadingOptions);
 ORT_DEFINE_RELEASE(IoBinding);
 ORT_DEFINE_RELEASE(ArenaCfg);
+ORT_DEFINE_RELEASE(Status);
```

Comment:
This macro wasn't friendly for C++ user, considering refactoring them as template inline function as well?